docs/source/index.rst

```diff
@@ -2,96 +2,99 @@ Welcome to the torchao Documentation
-        :link: dtypes.html
-        :link-type: url
+      .. grid-item-card:: :octicon:`file-code;1em`
+         Concepts
+         :img-top: _static/img/card-background.svg
+         :link: dtypes.html
+         :link-type: url
 
-        Learn about the key torchao concepts such
-        as dtypes, quantization, sparsity, among others.
+         Learn about the key torchao concepts such
+         as dtypes, quantization, sparsity, among others.
 
-     .. grid-item-card:: :octicon:`file-code;1em`
-        API Reference
-        :img-top: _static/img/card-background.svg
-        :link: api_ref_intro.html
-        :link-type: url
+      .. grid-item-card:: :octicon:`file-code;1em`
+         API Reference
+         :img-top: _static/img/card-background.svg
+         :link: api_ref_intro.html
+         :link-type: url
 
-        A comprehensive reference for the torchao
-        API and its functionalities.
+         A comprehensive reference for the torchao
+         API and its functionalities.
 
-Tutorials
-~~~~~~~~~
+   Tutorials
+   ~~~~~~~~~
 
-Ready to experiment? Check out some of the
-torchao tutorials.
+   Ready to experiment? Check out some of the
+   torchao tutorials.
 
-.. customcardstart::
+   .. customcardstart::
 
-.. customcarditem::
-   :header: Template Tutorial
-   :card_description: A placeholder template for demo purposes
-   :image: _static/img/generic-pytorch-logo.png
-   :link: tutorials/template_tutorial.html
-   :tags: template
+   .. customcarditem::
+      :header: Template Tutorial
+      :card_description: A placeholder template for demo purposes
+      :image: _static/img/generic-pytorch-logo.png
+      :link: tutorials/template_tutorial.html
+      :tags: template
 
-.. customcardend::
+   .. customcardend::
 
 
 .. ----------------------------------------------------------------------
 .. Below is the toctree i.e. it defines the content of the left sidebar.
 .. Each of the entry below corresponds to a file.rst in docs/source/.
 .. ----------------------------------------------------------------------
 
-.. toctree::
-   :glob:
-   :maxdepth: 1
-   :caption: Getting Started
-   :hidden:
+..
+   .. toctree::
+      :glob:
+      :maxdepth: 1
+      :caption: Getting Started
+      :hidden:
 
-   overview
-   getting-started
+      overview
+      getting-started
 
-.. toctree::
-   :glob:
-   :maxdepth: 1
-   :caption: Concepts
-   :hidden:
+   .. toctree::
+      :glob:
+      :maxdepth: 1
+      :caption: Concepts
+      :hidden:
 
-   dtypes
-   quantization
-   sparsity
-   performant_kernels
+      dtypes
+      quantization
+      sparsity
+      performant_kernels
 
-.. toctree::
-   :glob:
-   :maxdepth: 1
-   :caption: Tutorials
-   :hidden:
+   .. toctree::
+      :glob:
+      :maxdepth: 1
+      :caption: Tutorials
+      :hidden:
 
-   tutorials/template_tutorial
+      tutorials/template_tutorial
 
 .. toctree::
    :glob:
    :maxdepth: 1
    :caption: API Reference
-   :hidden:
 
    api_ref_intro
```

Comment:
In the api_ref_intro.rst file need to change:
```
``torchao`` API Reference
=========================
```
to
```
torchao API Reference
====================
```
So that link doesn't appear in blue: https://docs-preview.pytorch.org/pytorch/ao/195/index.html